### PR TITLE
SOTM

### PR DESCRIPTION
**Pull request by Mason-Beard**

### CODE REVIEW

File: src/main/java/frc/robot/commands/shooter/ShootNStrafe.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        MathUtil.applyDeadband(Math.abs(m_throttleInput.getAsDouble()), 0.05)
+            * Math.signum(m_throttleInput.getAsDouble());
+    double strafe =
+        MathUtil.applyDeadband(Math.abs(m_strafeInput.getAsDouble()), 0.05)
+            * Math.signum(m_strafeInput.getAsDouble());
+    double rotation =
+        MathUtil.applyDeadband(Math.abs(m_rotationInput.getAsDouble()), 0.05)
+            * Math.signum(m_rotationInput.getAsDouble());
+
+    //    if (DriverStation.isFMSAttached()
```
Comment (SherazF) [inline src/main/java/frc/robot/commands/shooter/ShootNStrafe.java:74]: Why is this here?

File: src/main/java/frc/robot/commands/swerve/SetSwerveDrive.java [hunk 2]
```diff
@@ -37,6 +37,8 @@ public void initialize() {}
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
+
+    // TODO: If you change value in here, change value in ShootNStrafe
```
Comment (SherazF) [inline src/main/java/frc/robot/commands/swerve/SetSwerveDrive.java:41]: Is there a way we can make both change if one changes? In case someone doesn't change both variables?

File: src/main/java/frc/robot/subsystems/Shooter.java [hunk 3]
```diff
@@ -13,6 +13,7 @@ public class Shooter extends SubsystemBase {
   private final TalonFX flywheelmotor1 = new TalonFX(CAN.flywheel1);
   private final TalonFX flywheelmotor2 = new TalonFX(CAN.flywheel2);
   private double m_rpm;
+  private double m_headingOffset;
```
Comment (SherazF) [inline src/main/java/frc/robot/subsystems/Shooter.java:16]: Where is this used?

File: src/main/java/frc/robot/RobotContainer.java [hunk 4]
```diff
@@ -97,6 +98,17 @@ private void configureBindings() {
...
+    xboxController
+        .x()
+        .whileTrue(
+            new ShootNStrafe(
+                m_swerveDrive,
+                m_vision,
+                () -> leftJoystick.getRawAxis(1),
+                () -> leftJoystick.getRawAxis(0),
+                () -> rightJoystick.getRawAxis(0),
+                600));
```
Comment (SherazF) [inline src/main/java/frc/robot/RobotContainer.java:111]: Why 600 specifically?

Comment (Mason-Beard) [reply src/main/java/frc/robot/RobotContainer.java:111]: placeholder (other than 420.69 for reasons)

File: src/main/java/frc/robot/subsystems/Shooter.java (hunk 3 above)
Comment (Mason-Beard) [reply src/main/java/frc/robot/subsystems/Shooter.java:16]: used to offset 2nd flywheel (only if we have 2 side flywheels)

File: src/main/java/frc/robot/commands/shooter/ShootNStrafe.java (hunk 1 above)
Comment (Mason-Beard) [reply src/main/java/frc/robot/commands/shooter/ShootNStrafe.java:74]: so we can drive and shoot at same time

File: src/main/java/frc/robot/commands/swerve/SetSwerveDrive.java (hunk 2 above)
Comment (Mason-Beard) [reply src/main/java/frc/robot/commands/swerve/SetSwerveDrive.java:41]: will work on soon, this is primarily so we can test the concept incase something else braakes :(
